eclair-core/src/main/scala/fr/acinq/eclair/db/PaymentsDb.scala

```diff
@@ -210,13 +229,15 @@ trait PaymentsOverviewDb {
  */
 sealed trait PlainPayment {
   val paymentHash: ByteVector32
+  val paymentType: Option[String]
   val paymentRequest: Option[String]
   val finalAmount: Option[MilliSatoshi]
   val createdAt: Long
   val completedAt: Option[Long]
 }
 
 case class PlainIncomingPayment(paymentHash: ByteVector32,
+                                paymentType: Option[String],
```

Comment:
couldn't we make `paymentType` required and manage compatibility within the db migration script (e.g. based on the description)?

Reply:
Oups I missed it here, paymentType is required and the migration is there.

Reply:
Fixed in `375b342`